src/game-mode.js

```diff
@@ -0,0 +1,207 @@
+import { words, FRAGMENT, btnRepeatImg } from "./variables.js";
+import { createBtn, createTitle, createImg } from "./create-elements.js";
+
+import { addPlayMode } from "./change-mode.js";
+import { setNumberOfGuess, setNumberOfMistakes } from "./localstorage.js";
+
+let repeatTrackValue = "";
+export let countCorrect = 0;
+export let countWrong = 0;
+export let stateOfGame = "";
+
+export function playQueue(event) {
+  let arr = createRandomQueue();
+  let arrShake = shake(arr);  
+  createRepeatBtn();
+  playSound(arrShake);
+}
+
+export function playSound(arr) {
+  let index = 0;
+  let clickedValue = "";
+  let right = new Audio("./assets/audio/correct.mp3");
+  let wrong = new Audio("./assets/audio/error.mp3");
+  let name = playMode(index, arr, countCorrect, countWrong);
+  let currentGame = document.querySelectorAll("div.flip-card.hide");
+  currentGame.forEach((element) => {
+    element.addEventListener("click", function chooseCard(event) {
+      let result = new Promise(function (resolve, reject) {
+        let target = event.currentTarget;
+        let value = target.firstChild;
+        clickedValue = value.dataset.id;
+        if (clickedValue === name) {          
+          setNumberOfGuess(clickedValue);
+          target.classList.add("correct");
+          target.removeEventListener("click", chooseCard);
+          countCorrect++;          
+          index++;
+          let field = document.querySelector("#answers");
+          field.classList.add("show");          
+          let correctImg = createImg(
+            "img",
+            "./assets/images/right.png",
+            "right"
+          );
+          clip();
+          field.append(correctImg);
+          right.play();
+          resolve();
+        } else {
+          setNumberOfMistakes(name);
+          countWrong++;          
+          let field = document.querySelector("#answers");
+          field.classList.add("show");          
+          let wrongImg = createImg("img", "./assets/images/wrong.png", "wrong");
+          clip();
+          FRAGMENT.append(wrongImg);
+          field.append(FRAGMENT);
+          wrong.play();
+        }
+      });
+      result.then(() => {
+        name = playMode(index, arr, countCorrect, countWrong);
+      });
+    });
+  });
+}
+
+function clip() {
+  let answContainer = document.querySelector("#answers");
+  let elements = answContainer.childNodes;
+  let value = elements.length;
+  let containerVolume = 5;  
+  if (value > containerVolume) {
+    elements[0].remove();
+  }
+}
+
+export function playMode(index, arr, countCorrect, countWrong) {
+  if (index === arr.length) {
+    stateOfGame = "end";
+    if (countWrong === 0 && stateOfGame === "end") {
```

Comment:
discuss
```suggestion
    if (countWrong === 0 && stateOfGame === "end") {
```
const value = countWrong === 0 && stateOfGame === "end" ? "success" : "failure";
showResult(value);